server/controllers/emptyBuildings/post.js

```diff
@@ -3,7 +3,9 @@ const { schemaBuildings } = require('../../validation');
 
 const postEmptyBuilding = async (req, res, next) => {
   try {
-    const newBuild = await schemaBuildings.validate(req.body);
+    const newBuild = await schemaBuildings.validate(req.body, {
+      abortEarly: false,
+    });
     await reportBuilding(newBuild);
     res.status(201).send({
       statusCode: 201,
```

Comment:
Receive the thumbnail and save it in the `uploads` dir, and then append its name associated with a timestamp to the object